### DISCUSSION TITLE
Controller in Matlab can not be used

### DISCUSSION THREAD

**Discussion by huiduan-25** [cyberbotics/webots#5600] [Q&A]
Hello,

I create a bipedal robot and I would like to use MATLAB to control two rotational motors. The  code for controller is simple but some warning just show up. 

Does anyone know how to solve this?

WARNING: Unable to find the 'matlab' executable in the current PATH. Please check your matlab installation. It should be possible to launch matlab from a terminal by typing 'matlab'. It may be necessary to add the matlab bin directory to your PATH environment variable. More information about the matlab installation is available in Webots' User guide.

Thanks

**Comment by omichel** [comment]
We are about to close this Q&A discussion from GitHub and move it to [robotics.stackexchange.com](https://robotics.stackexchange.com). Could you please post your question there, so that we can answer you? Please mention the [webots](https://robotics.stackexchange.com/questions/tagged/webots) tag in your question.
